apps/hosts/views.py

```diff
@@ -158,9 +158,7 @@ def challenge_host_list(request, challenge_host_team_pk):
 @authentication_classes((ExpiringTokenAuthentication,))
 def challenge_host_detail(request, challenge_host_team_pk, pk):
     try:
-        challenge_host_team = ChallengeHostTeam.objects.get(
-            pk=challenge_host_team_pk
-        )
+        challenge_host_team = ChallengeHostTeam.objects.get(pk=challenge_host_team_pk)
```

Comment:
Can we please use `get_challenge_host_team_model` here. (If it is not present then can we please create one in the utils file similar to `get_challenge_model`)